hal/armv7m/imxrt/117x/peripherals.h

```diff
@@ -247,20 +249,20 @@
 
 /* FLASH */
 
-#define FLASH_NO                  (FLASH_FLEXSPI1_MOUNTED + FLASH_FLEXSPI2_MOUNTED)
+#define FLASH_NO                  FLEXSPI_COUNT
```

Comment:
Previously `FLASH_FLEXSPI1_MOUNTED` and `FLASH_FLEXSPI2_MOUNTED` were used to determine which instances are used in a project. On non-standard/custom boards it is possible to have only `FLASH_FLEXSPI2_MOUNTED` instance and no `FLASH_FLEXSPI1_MOUNTED`, so if `FLEXSPI_COUNT=1` is set, you then always start with `FLEXSPI1` and not `FLEXSPI2`. Previous idea was more flexible from my point of view.

Reply:
I agree that it would be nice to have this feature however I don't think it was ever there. FLASH_*_MOUNTED defines were used only to set FLASH_NO. There was never support for using FLEXSPI2 memories only.